shared-helpers/src/auth/AuthContext.ts

```diff
@@ -179,14 +180,18 @@ const reducer = createReducer(
 )
 
 export const AuthContext = createContext<Partial<ContextProps>>({})
-export const AuthProvider: FunctionComponent = ({ children }) => {
+export const AuthProvider: FunctionComponent<{ isTesting?: boolean }> = ({
+  children,
+  isTesting = false, // Set this to true for tests that you need to set the user
```

Comment:
Totally fine if this is the best case, but would prefer to not add testing paths into app code

Reply:
I also would prefer to not do this. I might tinker around on the test some more to see if there is a better way

Reply:
I came up with a solution to mock out the session storage. I like it better since it doesn't touch the AuthContext anymore.